9c-internal/multiplanetary/network/heimdall.yaml

```
@@ -90,6 +90,28 @@ dataProvider:
   nodeSelector:
     eks.amazonaws.com/nodegroup: heimdall-internal-r7g_l_2c
 
+testDataProvider:
+  enabled: true
+
+  render: true
+  migrateDatabase: false
```

```suggestion
  migrateDatabase: true
```

For inital db setup.